joomla/joomla-cms
# Add the End of Life (EOL) plugin for Joomla! 3.10

Pull Request for Issue # .

### Summary of Changes
Updated language files
Updated quicklink plug to add new conditions, add a quicklinks group and restore the warning notice

### Testing Instructions
use a 3.9 site and set the updater to custom url. Add the github url and see that the update works and that the plugin is updated
Do a fresh install from the github install and see that the plugin is there.

### Actual result BEFORE applying this Pull Request
The Quicklinks was not in its own group but more hidden. 
There was a red image which has been replaced with the icon for information/warning
No warning across the top

### Expected result AFTER applying this Pull Request
The warning message across the top has been restored as in Joomla 2.5 EOS plugin to make it more obvious and allow a place to show different messages over time if needed
The Quicklinks is in its own group and has the icon added and the warning class added.

If the date has passed and the version is 3.10 and its the administrator control panel you should see this


You can simulate this by editing plugins->quicklink->eosnotify->eosnotify.php
 line 52 in your instillation 
if (!$this->app->isClient('administrator') || version_compare(JVERSION, '3.10', '>=') || Factory::getDate()->toSql() <= '2021-04-15')
and change the date 2021-04-15 to 2021-01-01
to make the plugin think we have passed the nominated date (my birthday) 

![eos](https://user-images.githubusercontent.com/1078933/103467087-40308000-4d43-11eb-8473-afe995204dec.PNG)


### Documentation Changes Required

@zero-24
Please take a look at this draft which I can then add to depending on the outcome of our meeting.

@bayareajenn
Was this more what you had in mind and do we know where the documentation will be held that it needs to link to?
